server/src/dataSources/__tests__/joinEventAPI.js

```diff
@@ -0,0 +1,72 @@
+const {
```

Comment:
Change filename so it follows this guideline: File names must be all lowercase and may include underscores (_) or dashes (-), but no additional punctuation. Follow the convention that your project uses. Filenames’ extension must be .js.


Same for all below files